po/r-novice-gapminder.ja.po

```diff
@@ -495,44 +495,44 @@ msgid ""
 msgstr ""
 "All Software Carpentry and Data Carpentry instructional material is\n"
 "made available under the [Creative Commons Attribution\n"
-"license][cc-by-human]. The following is a human-readable summary of\n"
+"license](https://creativecommons.org/licenses/by/4.0/deed.ja). The following is a human-readable summary of\n"
 "(and not a substitute for) the [full legal text of the CC BY 4.0\n"
-"license][cc-by-legal]."
+"license](https://creativecommons.org/licenses/by/4.0/legalcode.ja)."
```

Comment:
These lines also need to be translated

Reply:
Yes, these lines as well. These were added by Software Carpentry and aren't in the human readable license (so we're free to translate as we wish). https://github.com/swcarpentry-ja/i18n/pull/127/files#diff-627559538775bc873d27f74c80ae4d17L546-L547

Reply:
It would be good to get input from people with better Japanese than on this one. 😄

Reply:
@masamiy has taken care of this.